Library/Formula/v8.rb

```diff
@@ -20,6 +22,8 @@ class V8 < Formula
   depends_on :macos => :lion
 
   depends_on :python => :build # gyp doesn't run under 2.6 or lower
+  depends_on "pkg-config" => :build
+  depends_on "icu4c"
```

Comment:
Hrm... if you insist. My rational was:

* ICU support is built by default in stock v8
* Other non-brew packages that use v8 support it (Node, Chromium)

I also ran into probs doing `if build.with? "icu4c"` when I tried, but that's just me being a noob.